lib/harvester.js

```diff
@@ -433,6 +453,7 @@ Harvester.prototype._removeRoutes = function (name, methods, routes) {
 
   this.adapter.awaitConnection().then(function () {
     (methods || []).forEach(function (verb) {
+//        TODO imho this method is badly implemented. `router` does not have `routes` property.
```

Comment:
So what do we do with this code? It's being used by `noIndex` which I is not even tested.